atom/atom.py

```diff
@@ -194,7 +203,7 @@ def __new__(meta, name, bases, dct):
         post_validates = []         # Post validate methods: _post_validate_*
         seen_sentinels = set()      # The set of seen sentinels
         seen_decorated = set()      # The set of seen @observe decorators
-        for key, value in dct.iteritems():
+        for key, value in dct.items():
```

Comment:
This is debatable. I prefer to have a Python 3 oriented code base. Actually a similar choice was done in Chaco recently.